node/out_response.js

```diff
@@ -234,9 +281,29 @@ TChannelOutResponse.prototype.sendNotOk = function sendNotOk(res1, res2) {
 TChannelOutResponse.prototype.send = function send(res1, res2) {
     var self = this;
 
+    self._emitLatency();
     self.arg2 = res1;
     self.arg3 = res2;
 
+    if (self.ok) {
+        self.channel.inboundCallsSuccessStat.increment(1, {
+            'calling-service': self.callingService,
+            'service': self.serviceName,
+            'endpoint': self.endpoint
+        });
+    } else {
+        var type = [self.arg2, self.arg3].filter(
+            function filter(val) {
+                return val;
+            }).join('.');
+        self.channel.inboundCallsAppErrorsStat.increment(1, {
+            'calling-service': self.callingService,
+            'service': self.serviceName,
+            'endpoint': self.endpoint,
+            'type': type
```

Comment:
default this to `'unknown'` We need some kind of `outResponse.setErrorType()` method to support this properly.